src/components/common/TechnologyLogos/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import withStyles from 'react-jss';
+import cx from 'classnames';
+import { withPrefix } from 'gatsby-link';
+
+import technologyBrands from '../../../data/technologyBrands.json';
+
+import styles from './styles';
+
+const TechnologyLogos = props => {
+  const { classes } = props;
```

Comment:
Keep the convention to make flexible components. So the properties would be like
```js
const { theme, sheet, classes, className, ...etc } = props;
const cls = cx(classes.root, className);
...
```